app/controllers/api/v1/inbox_members_controller.rb

```diff
@@ -1,49 +1,52 @@
-class Api::V1::InboxMembersController < Api::BaseController
-
-  before_action :fetch_inbox, only: [:create, :show]
-  before_action :current_agents_ids, only: [:create]
-
-  def create #update also done via same action
-    #get all the user_ids which the inbox currently has as members.
-    #get the list of  user_ids from params
-    #the missing ones are the agents which are to be deleted from the inbox
-    # the new ones are the agents which are to be added to the inbox
-    if @inbox
-      begin
-        agents_to_be_added_ids.each do |user_id|
-          @inbox.add_member(user_id)
+module Api
```

Comment:
@ancarosu  Can you add test cases for this. We are getting a lot of PRs these days. Its high time to add tests so that we can move fast.

Reply:
Okay, will do!